test/types/index.ts

```diff
@@ -2,12 +2,13 @@
 // This file will be passed to the TypeScript CLI to verify our typings compile
 
 import * as fastify from '../../fastify'
-import * as cors from 'cors'
 import * as http from 'http';
 import * as http2 from 'http2';
 import { readFileSync } from 'fs'
 import { createReadStream, readFile } from 'fs'
 
+const cors = require('cors');
```

Comment:
Can you add a comment on why this is done in this way?

Reply:
I'll add a comment but this test is not really doing anything at this point since `cors` is typed as an `any`...